src/index.js

```diff
@@ -0,0 +1,45 @@
+import './style.css';
+import '@fortawesome/fontawesome-free/js/fontawesome';
+import '@fortawesome/fontawesome-free/js/solid';
+import '@fortawesome/fontawesome-free/js/regular';
+import '@fortawesome/fontawesome-free/js/brands';
+
+const toDoList = document.querySelector('.to-do-list');
+const listItems = [
+  {
+    description: 'Feed the dog',
+    completed: false,
+    index: 0,
+  },
+  {
+    description: 'Make breakfast',
+    completed: false,
+    index: 1,
+  },
+  {
+    description: 'Get grocery',
+    completed: false,
+    index: 2,
+  },
+];
+
+listItems.forEach((item) => {
+  const newItem = document.createElement('li');
```

Comment:
- [ ] Kindly make sure that the tasks are rendered in order of their indices values, as per [this](https://github.com/microverseinc/curriculum-javascript/blob/main/todo-list/m1_list_structure.md#:~:text=The%20list%20should%20appear%20in%20order%20of%20the%20index%20values%20for%20each%20task.) requirement.